File: Python3/Secuenciales/ej11/ej11.py
# -*- coding: utf-8 -*-

#Pide al usuario dos números y muestra la "distancia" entre ellos 
#(el valor absoluto de su diferencia, de modo que el resultado sea siempre positivo).

#1.pedir numero1
#2.almacenar numero1
#3.pedir numero2
#4.almacenar numero2
#5.obtener el número mayor
#6.calcular distancia mediante la fórmula: abs(max-min)

print("..::Distancia entre dos números::..")
numero1 = float(input("Introduce número1> "))
numero2 = float(input("Introduce número2> "))
if(numero1>numero2):
	max = numero1
	min = numero2
elif(numero2>numero1):
	max = numero2 
	min = numero1
else:
	print("Ambos son el mismo número")
print("La distancia entre "+str(numero1)+" y "+str(numero2)+" es "+str(abs(max-min)))

File: Java/Secuenciales/ej11.java
import java.util.Scanner;

/**
* Pide al usuario dos números y muestra la "distancia" entre ellos 
* (el valor absoluto de su diferencia, de modo que el resultado sea siempre positivo).
*/

//Almacenar numero1
//Almacenar numero2
//distancia = Math.abs(numero1-numero2)
//mostrar distancia

public class ej11 {
	public static void main(String[] args) {
		double numero1;
		double numero2;
		double distancia;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("Calcular la distancia entre dos números");
		System.out.println("Introduce el primer número> ");
		numero1 = scanner.nextDouble();
		System.out.println("Introduce el segundo número> ");
		numero2 = scanner.nextDouble();
		
		distancia = Math.abs(numero1-numero2);
		
		System.out.println("La distancia entre "+numero1+" y "+numero2+" es "+distancia);
	}
}